lib/changeful/helpers/view_helpers.rb

```
@@ -26,7 +26,9 @@ module ViewHelpers
     # 
     # @return [String] a content based on a key
     def changeful_content(key, options = {})
-      contents ||= Content.all_contents_in(current_view)
+      current_view = get_file_path
+
+      contents ||= Content.all_contents_in(current_view) if current_view.present?
```

is there any case where current_view will be not be present?
If it is not present, maybe we should throw an exception or just return false?
We can make this method code shorter and clearer.